src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,4 @@
+package duke;
+
+public class DukeException extends Exception {
```

Comment:
since it is a custom exception, you need to provide Javadoc comments to tell why it is used